index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Material+Icons+Outlined">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="mob.css">
+    <link rel="stylesheet" href="about.css">
+    <link rel="stylesheet" media="(min-width:768px)" href="aboutDesktop.css">
+    <link rel="stylesheet" media="(min-width:768px)" href="des.css">
+    <title>Capstone1 project</title>
+</head>
+<body>
+    <header>
+        <div class="top">
+            <div class="media">
+                <ul class="mediaList">
+                    <li>
+                        <div class="mediaIcon">
+                            <a class="face" href="#"><img src="images/png/facebook.png"  alt="Facebook "></a>
+                        </div>
+                    </li>
+                    <li>
+                        <div class="mediaIcon twit">
+                            <a href="#"><img src="images/png/twitter.png" alt="Twitter icon"></a>
+                        </div>
+                    </li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+            </div>
+            <div class="desktopBar">
+                <div class="logo">
+                    <a href="#"><img src="images/png/logo1.png"  height="75px" alt="sonu logo icon"></a>
+                </div>
+                <ul class="deskOptions">
+                    <li>
+                        <a href="about.html" class="deskLi">About</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">Program</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">Join</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">Sponsors</a>
+                    </li>
+                    <li>
+                        <a href="#" class="deskLi">contact</a>
+                    </li>
+                </ul>
+                <a class="rock deskLi" href="#">Sony Music</a>
+            </div>
+        </div>                                                                               
+        <div class="access">
+            <a class="openMenu action" href="#"><span class="material-icons-outlined open">menu</span></a>
+        </div>
+        <!-- Mobile menu list -->
+        <div class="mobileMenu">
```

Comment:
- [ ] There is an overflow that is causing the positioning of themobile version to break, kindly fix it

Your design
![image](https://user-images.githubusercontent.com/81932478/205454781-8ab5e0c3-06a7-45ec-b05c-7e626b3d1d53.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205454807-6982e048-093c-4192-ab2e-b47011a9cb17.png)